rosa_planning/rosa-planning-limits.adoc

```diff
@@ -0,0 +1,11 @@
+include::modules/attributes-openshift-dedicated.adoc[]
+
+[id="rosa-planning-limits-and-scalability"]
+= Planning limits and scalability
```

Comment:
Friendly reminder that this title doesn't match the value in the topic map file.